### PR TITLE
[Konvoy COPS 7154] Backporting 2.2 air-gapped bundle changes to 2.1

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://jira.d2iq.com/browse/COPS-7154

## Description of changes being made

Description in title, resolves ticket.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4358.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cmastr — APPROVED** [review]

**Comment by msdolbey** [comment]
Links need to be changed from 2.2.0 to 2.1.1

**Comment by msdolbey** [comment]
So, I checked the links that are in here - and they don't seem to be valid in the same way. 
We might need to confirm with someone on COPS or engineering. 
FYI @dkoshkin 

Example, I tried this: 
`curl -O https://downloads.d2iq.com/dkp/v2.1.1/konvoy-bootstrap_v2.1.1.tar`

and it isn't really a valid file for me (compared to 2.2.0 of the same file)

File: pages/dkp/konvoy/2.1/image-builder/create-azure-image/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+-   Download the latest [Konvoy Image Builder](https://github.com/mesosphere/konvoy-image-builder/releases) bundle (prefixed with `konvoy-image-bundle`) for your OS. Do not use the release prefixed with `konvoy-image-builder`.
+
+-   Create a working `Docker` setup.
+
+## Extract the bundle
+
+Extract the bundle and `cd` into the extracted `konvoy-image-bundle-$VERSION_$OS` folder. The bundled version of `konvoy-image` contains an embedded `docker` image that contains all the requirements for building.
+
+<p class="message--note"><strong>NOTE: </strong> The <code>konvoy-image</code> binary and all supporting folders are also extracted. When run, <code>konvoy-image</code> bind mounts the current working directory (<code>${PWD}</code>) into the container to be used.</p>
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.1/image-builder/create-azure-image/index.md:25]: @mwrightd2 
I don't know if Azure is valid to get built on 2.1.1.

I know it previously wasn't able to, but maybe the upgrades in KIB independently can make it work, but we may have to axe this.

What do you think @faiq ?

**Comment by gflorinskaya** [comment]
> Example, I tried this: `curl -O https://downloads.d2iq.com/dkp/v2.1.1/konvoy-bootstrap_v2.1.1.tar`
> 
> and it isn't really a valid file for me (compared to 2.2.0 of the same file)

link currently does not exist. I assume it will be once it is released, right? @faiq

File: pages/dkp/konvoy/2.1/image-builder/create-azure-image/index.md (hunk 1 above)
Comment (faiq) [reply pages/dkp/konvoy/2.1/image-builder/create-azure-image/index.md:25]: I think its OK to leave it in and specify we need to use a v1.12.0 of KIB to build the images

**Comment by faiq** [comment]
> link currently does not exist. I assume it will be once it is released, right? @faiq

yep! that artifact will be up there when the release is ready :)

**Comment by mwrightd2** [comment]
Links have been flipped and sections have been edited @dkoshkin @msdolbey

File: pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/new/index.md [hunk 2]
```diff
@@ -64,7 +79,9 @@ enterprise: false
     --internal-load-balancer=true \
     --additional-security-group-ids=${AWS_ADDITIONAL_SECURITY_GROUPS} \
     --registry-mirror-url=${DOCKER_REGISTRY_ADDRESS} \
-    --registry-mirror-cacert=${DOCKER_REGISTRY_CA}
+    --registry-mirror-cacert=${DOCKER_REGISTRY_CA} \
+    --registry-mirror-username=${DOCKER_REGISTRY_USERNAME} \
+    --registry-mirror-password=${DOCKER_REGISTRY_PASSWORD}
```
[the remark this answers is not in the document]
Comment (dkoshkin) [reply pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/new/index.md:84]: Please commit ☝️

File: pages/dkp/konvoy/2.1/image-builder/create-azure-image/index.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---
+layout: layout.pug
+navigationTitle: Create Azure Image
+title: Create a custom Azure Image
```
[the remark this answers is not in the document]
Comment (mwrightd2) [reply pages/dkp/konvoy/2.1/image-builder/create-azure-image/index.md]: Deleted!

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-cluster/index.md [hunk 4]
```diff
@@ -8,25 +8,84 @@ beta: false
...
+    configmap/tigera-operator-preprovisioned-example created
+    clusterresourceset.addons.cluster.x-k8s.io/local-volume-provisioner-preprovisioned-example created
+    configmap/local-volume-provisioner-preprovisioned-example created
+    clusterresourceset.addons.cluster.x-k8s.io/node-feature-discovery-preprovisioned-example created
+    configmap/node-feature-discovery-preprovisioned-example created
+    clusterresourceset.addons.cluster.x-k8s.io/nvidia-feature-discovery-preprovisioned-example created
+    configmap/nvidia-feature-discovery-preprovisioned-example created
+    clusterresourceset.addons.cluster.x-k8s.io/metallb-preprovisioned-example created
+    configmap/metallb-installation-preprovisioned-example created
+    ```
```
[the remark this answers is not in the document]
Comment (mwrightd2) [reply pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-cluster/index.md]: Removed!

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/prerequisites-airgapped/index.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
+---
```
Comment (dkoshkin) [inline pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/prerequisites-airgapped/index.md:1]: These instructions need to be updated the same way pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/create-an-ami/index.md was.
Different URL links, removing non-existent bundles, etc.

File: pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/seed-a-registry/index.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+1.  Download the images bundle.
+
+    ```bash
+    curl -o konvoy-image-bundle.tar.gz -O http://downloads.d2iq.com/konvoy/airgapped/v2.1.1/konvoy_image_bundle_v2.1.1_linux_amd64.tar.gz
+    ```
+
+1.  Place the bundle in a location where you can load and push the images to your private docker registry.
+
+
```
Comment (dkoshkin) [inline pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/seed-a-registry/index.md:22]: This is missing the shell script from step 3 https://docs.d2iq.com/dkp/kommander/2.1/install/air-gapped/#load-the-docker-images-into-your-docker-registry

**Pull request merged**